sfcasts/2019_paris/lightning-talks.md

```diff
@@ -5,3 +5,50 @@ Symfony Live Paris 2019 lightning talks by various authors.
 This talk is in French and (sorry!) no transcript is available.
 ***
 
+## Les side-projects, c'est cool ! (@pierstoval)
```

Comment:
Links to Twitter or GitHub would be nice for the speakers

Reply:
Done :)